pkg/controller/expcertificates/issuing/issuing_controller.go

```diff
@@ -267,7 +279,10 @@ func (c *controller) ProcessItem(ctx context.Context, key string) error {
 		return c.issueCertificate(ctx, log, nextRevision, crt, req, pkData)
 	}
 
-	// issueTemporaryCertificate
+	// Issue temporary certificate if needed
+	if err := c.ensureTemporaryCertificate(ctx, crt, pkData); err != nil {
+		return err
+	}
```

Comment:
If we issue a temporary certificate, should we return early and wait to observe the Secret changing before continuing? If the CertificateRequest is Ready, we'll quite likely get a resourceVersion conflict further on as the lister may not have observed the new Secret resource